e2e/internal/e2e/priv_linux.go

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package e2e
+
+/*
```

Comment:
Any reason why the code that is commented out is kept?

Reply:
This is the CGO part